nest-forms-backend/src/nases/nases.service.ts

```diff
@@ -330,6 +330,16 @@ export default class NasesService {
       )
     }
 
+    // Check if attachments are scanned and safe
+    const formAttachmentsReady =
+      await this.filesService.areFormAttachmentsReady(id)
+    if (!formAttachmentsReady.filesReady) {
+      throw this.throwerErrorGuard.ForbiddenException(
+        NasesErrorsEnum.UNABLE_TO_SEND,
```

Comment:
Maybe it would be nice to have separate state for this, let's say `NasesErrorsEnum.SCAN_DIDNT_FINISH`.